osu.Game/Online/Chat/DrawableLinkCompiler.cs

```diff
@@ -52,6 +62,32 @@ private void load(OsuColour colours)
 
         protected override IEnumerable<Drawable> EffectTargets => Parts;
 
+        public MenuItem[] ContextMenuItems
+        {
+            get
+            {
+                List<MenuItem> items = new List<MenuItem>();
+                bool hasTooltip = TooltipText.ToString() != "";
+
+                string text = hasTooltip ? TooltipText.ToString() : GetUrlFromPart(Parts);
+
+                bool isChannelorLobbyLink = text.Contains("osu://chan") || text.Contains("osump://");
+
+                if (!isChannelorLobbyLink)
+                {
+                    items.Add(new OsuMenuItem("Open", MenuItemType.Highlighted, () => host.OpenUrlExternally(text)));
+                }
+
+                items.Add(new OsuMenuItem("Copy URL", MenuItemType.Standard, () =>
```

Comment:
This doesn't work on wiki.

![osu!_OcsC28PbZo](https://user-images.githubusercontent.com/35318437/235284993-50ac64f5-84a3-40e0-b912-56875d1b42c1.gif)

Reply:
Good catch, it seems like the many of the wiki links have different TooltipText values compared to the displayed hover tooltip. I specifically used TooltipText because it was an easy solution for the /np scenario, making the assumption that it should always correspond to the url. 
Is there a distinction between the two?

Reply:
Im partway through an implementation that uses Link to derive LinkAction and Url to help with this case. I can also pass Link through LinkHandler to also make the highlighted action behave like a left click as noted below.

However it seems like the Wiki clickables are not Links. Is this an osu-wiki dependency? I might be stuck here.